File: Iniciante/1038 - Lanche/C/1038 - Lanche.c
#include <stdio.h>

int main() {
	float valor;  // variáveis
	int cod, quant;
	
	scanf("%d %d", &cod, &quant); // entrada
	
	if(cod == 1) { // condicional
		valor = 4.00;
	
	} else if(cod == 2) {
		valor = 4.50;
	
	} else if(cod == 3) {
		valor = 5.00;
		
	} else if(cod == 4) {
		valor = 2.00;
		
	} else if(cod == 5) {
		valor = 1.50;
		
	}
	
	printf("Total: R$ %.2f\n", quant*valor); // cálculo + saída

	return 0;	
}

File: Iniciante/1038 - Lanche/Python/1038 - Lanche.py
# recebe o valor das variáveis na mesma linha
cod, qntd = input().split(' ')

# converte para o tipo int
cod = int(cod)
qntd = int(qntd)

# identificação do produto e seu preço
if cod == 1:
    preco = 4.00

elif cod == 2:
    preco = 4.50

elif cod == 3:
    preco = 5.00

elif cod == 4:
    preco = 2.00

elif cod == 5:
    preco = 1.50

# cálculo do total + saída
total = preco * qntd
print('Total: R$ {:.2f}'.format(total))